bodhi/server/schemas.py

```diff
@@ -403,6 +403,22 @@ class SaveReleaseSchema(CSRFProtectedSchema, colander.MappingSchema):
     )
 
 
+class UpdateCriticalPackages(colander.MappingSchema):
+    """
+    An API schema for critical path packages.
+
+    This schema is used by bodhi.server.services.releases.add_critpath_packages()
+    and bodhi.server.services.releases.remove_critpath_packages().
+    """
+
+    packages = Packages(
+        colander.Sequence(accept_scalar=True),
+        location='querystring',
```

Comment:
Shouldn't the location be the body? We probably don't want to use the querystring in POST and PUT calls to send the data.